examples/permissioned-token/test/failing-paths.js

```diff
@@ -588,7 +588,7 @@ describe('Permissioned Token Tests - Failing Paths', function () {
           sign: types.SignType.SecretKey,
           fromAccount: asaManager.account,
           appID: ctx.controllerappID,
-          payFlags: { totalFee: 2000 },
+          payFlags: { totalFee: 2000, flatFee: true },
```

Comment:
Could you please add a note about  using  `flatFee` in docs (i.e what the difference would be between using `{ totalFee: 2000 }` and `{ totalFee: 2000, flatFee: true }`) ?

Reply:
after the update where we set flatFee true if totalFee is set, there is no difference i will remove these